docker-compose.yml

```diff
@@ -128,6 +134,16 @@ services:
     volumes:
       - .:/app
       - npm-cache:/root/.npm
+    depends_on:
```

Comment:
How does this dependency contribute to the way that watch and refine containers are going to interact with each other?

Reply:
It's just for timing. With the dependency set up like that, the health check stuff built into Compose makes it wait for watch to start up before starting up refine. (It's sort of an off-label use of that; usually you'd use the health check to wait for a service (like your DB or redis or whatever) to come online before starting something that has to use it. In this case, watch and refine don't depend on each other but they do use the same bits of the filesystem so they'll bump heads otherwise.)